datashuttle/command_line_interface.py

```diff
@@ -98,12 +64,12 @@ def help(help_type):
     "datashuttle [PROJECT NAME] [COMMAND] [OPTIONS]"
     "\n\n"
     "To get detailed help for commands and their optional arguments, "
-    "\ntype python -m datashuttle [PROJECT NAME] [COMMAND] --help'"
+    "\ntype datashuttle [PROJECT NAME] [COMMAND] --help'"
     "\n\n"
     "On first use it is necessary to setup configurations. \ne.g."
-    "'python -m datashuttle [PROJECT NAME] make_config_file [OPTIONS]'"
+    "'datashuttle [PROJECT NAME] make_config_file [OPTIONS]'"
```

Comment:
use dashes instead of underscores
```suggestion
    "'datashuttle [PROJECT NAME] make-config-file [OPTIONS]'"
```